autopts/ptsprojects/stack/layers/gap.py

```diff
@@ -156,6 +160,48 @@ def wait_periodic_established(self, timeout):
 
         return False
 
+    def wait_big_established(self, timeout):
+        if self.big_sync_established_rxed:
+            return True
+
+        if wait_for_event(timeout, lambda: self.big_sync_established_rxed):
+            self.big_sync_established_rxed = False
+            return True
+
+        return False
+
+    def wait_bis_data_path_setup(self, bis_id, timeout):
```

Comment:
```suggestion
    def wait_bis_data_path_setup(self, bis_id, timeout=10):
```

Consider defining default timeouts so that each caller does not need to provide this.

Could also consider defaulting `bis_id=None`